docs/modules/configuration/pages/tracing.adoc

```diff
@@ -53,5 +59,6 @@ tracing:
   sampleProbability: 0.5
   exporter: otlp
   otlp:
+    protocol: grpc # or http
```

Comment:
I think we should use `grpc` or `http/protobuf` to be explicit and match the [OTLP specification](https://opentelemetry.io/docs/specs/otel/protocol/exporter/#specify-protocol).

Reply:
We use grpc maybe that's basic for usage

Reply:
Sorry, I mean that the valid values should be `"grpc"` or `"http/protobuf"` (not `"http"`), so that we match the specification.

```suggestion
    protocol: grpc # or http/protobuf
```

Reply:
Oh no problem, okay I understand now.

Reply:
the Alternatives are [grpc , http/protobuf] Am I correct?

Reply:
Yes, that's correct